packages/bits/src/lib/wizard-v2/wizard.directive.ts

```diff
@@ -116,6 +124,26 @@ export class WizardDirective extends CdkStepper implements OnChanges, AfterConte
         super.ngOnDestroy();
     }
 
+    public getStepState(i: number): StepState {
```

Comment:
This method is quite large and is called from the template in every wizard implementation. I'd consider reworking it in a way that returns static value instead of calculating it every time. (Including operations like `toArray` and linear search of `indexOf`)